EvolutionModel/Species.cs

```diff
@@ -1,5 +1,9 @@
-﻿namespace Evolution.Model {
-  public class Species {
+﻿using System.Linq;
+using System.Collections.Generic;
+using System;
+
+namespace Evolution.Model {
+  public class Species : IEater {
     public bool IsPredator { get; internal set; }
```

Comment:
Voor species weten we nog niet hoe we IsPredator willen implementeren. Voorstel is om daar nu `public bool IsPredator => throw new NotImplmentedException();` van te maken.

Reply:
De reden dat de compiler dit niet OK vindt voor de interface is omdat de setter internal was. Alle interface meuk is per definitie public. Maar omdat we `set` uit de interface gaan halen, is het handig om het ook hier weg te halen.